.github/scripts/notifier.py

```diff
@@ -5,8 +5,8 @@
 
 import requests
 
-ALL_DEVS = "1044950703795359756"
-BACKEND_DEVS = "1044951412674674758"
+ALL_DEVS = "735068019440615516"
+BACKEND_DEVS = "983289520000737330"
```

Comment:
Didn't notice but this one should also have single quotes

Reply:
Any reason why there is not some automated way to warn about this?

It seems kind of silly to manually enforce such a styling rule.
In the frontend we have the linter take care of it and we don't bother.